### PR TITLE
View project overlay

### PR DESCRIPTION
**Pull request by LancePTB**
NOTE: Check the 1st project on the index page to hover for the text and button stuff. I didn't nail the text on the project header on hover exactly, but the rest are okay. I'm going to apply this on the team section on the index page as well

### CODE REVIEW

File: public/app/index.html [hunk 1]
```diff
@@ -124,37 +124,36 @@
     </div> <!-- End Hero Block three -->
     <div class="herofive-block">
       <div class="herofive-block-content">
-        <h2>Our Recent Work</h2>
-        <div class='block'>
+        <h2>Our Recent Work</h2>  
+        <!-- <div class="block">     -->
```
Comment (akshat12) [inline public/app/index.html]: Delete commented code

File: public/app/index.js [hunk 2]
```diff
@@ -83,7 +83,13 @@ $(document).ready(function(){
       navbarOpen = true;
       $(this).addClass('navbar-toggle-open');
     }
-  })
+  });
+
+  // $('.project-hover').on('hover',function(){
+  //   $('.flyout').show();
+  // },function(){
+  //   $('.flyout').hide();
```
Comment (akshat12) [inline public/app/index.js]: Delete commented code

File: public/lib/css/index.css [hunk 3]
```diff
@@ -901,15 +901,92 @@ header a:hover {
...
     padding-top: 2.5em;
     padding-bottom: 2.5em;
   }
   .portfolio {
+    /*margin:0 20px;*/
     padding-top: 3em;
     padding-bottom: 3em;
+    width: 230px;
     float: left;
+    margin-right: 1.6949152542%;
```
Comment (akshat12) [inline public/lib/css/index.css:1046]: This margin seems a bit too specific. Where did that number come from?

Comment (LancePTB) [inline public/lib/css/index.css:1046]: This is present in our team section on the index page as well, this is pretty much from the compiled css that kevin gave us off from the yeoman-ed project. At first i wanted to use a manual margin (hence a margin declaration not stated above), but it's safe to use this as well.

**Pull request closed without merging**